jerry-main/main-unix.c

```diff
@@ -627,7 +627,7 @@ main (int argc,
 
   if (jerry_is_feature_enabled (JERRY_FEATURE_SNAPSHOT_EXEC))
   {
-    for (int i = 0; i < exec_snapshots_count; i++)
+    for (i = 0; i < exec_snapshots_count; i++)
```

Comment:
That for loop ends before this statement, I've just reuse the `i` variable here.